samples/charts/data-chart/series-marker-template/src/DataChartSeriesMarkerTemplate.tsx

```diff
@@ -0,0 +1,97 @@
+// data chart's elements for column series:
+import { IgrNumericYAxis } from 'igniteui-react-charts';
+import { IgrCategoryXAxis } from 'igniteui-react-charts';
+import { IgrColumnSeries } from 'igniteui-react-charts';
+// data chart's modules:
+import { IgrDataChart } from 'igniteui-react-charts';
+import { IgrDataChartCoreModule } from 'igniteui-react-charts';
+import { IgrDataChartCategoryModule } from 'igniteui-react-charts';
+import { IgrDataChartInteractivityModule } from 'igniteui-react-charts';
+import { DataTemplateMeasureInfo } from 'igniteui-react-core';
+import { DataTemplateRenderInfo } from 'igniteui-react-core';
+import * as React from 'react';
+
+IgrDataChartCoreModule.register();
+IgrDataChartCategoryModule.register();
+IgrDataChartInteractivityModule.register();
+
+export default class DataChartSeriesMarkerTemplate extends React.Component<any, any> {
+    public data: any[];
+
+    constructor(props: any) {
+        super(props);
+
+        this.state = { markersType: "Circle" }
+        this.initData();
+    }
+
+    public render(): JSX.Element {
+        return (
+            <div className="igContainer">
+                <div className="igComponent" style={{height: "calc(100% - 35px)"}} >
```

Comment:
remove this code because this sample does not have any options/settings for an user to interact with:
```style={{height: "calc(100% - 35px)"}}```